libs/coin-modules/coin-solana/src/signOperation.ts

```diff
@@ -52,7 +52,9 @@ function getResolution(
   transaction: Transaction,
   deviceModelId?: DeviceModelId,
 ): Resolution | undefined {
-  if (!transaction.subAccountId || !transaction.model.commandDescriptor) return;
+  if (!transaction.subAccountId || !transaction.model || !transaction.model.commandDescriptor) {
+    return;
+  }
```

Comment:
We should be able to revert this the model should always be present in signOperation now